replica/database.hh

```diff
@@ -2090,6 +2090,23 @@ public:
     // * the `locator::topology` instance corresponding to the passed `locator::token_metadata_ptr`
     //   must contain a complete list of racks and data centers in the cluster.
     void check_rf_rack_validity(const bool enforce_rf_rack_valid_keyspaces, const locator::token_metadata_ptr) const;
+
+    /// Verify that all existing materialized views are valid.
```

Comment:
The comment needs to be updated, it says that the function throws an exception but in fact it only prints a warning. Let's fix it in a follow-up.